librosa/feature/spectral.py

```diff
@@ -766,7 +766,7 @@ def spectral_flatness(y=None, S=None, n_fft=2048, hop_length=512,
 
 
 def rms(y=None, S=None, frame_length=2048, hop_length=512,
-        center=True, pad_mode='reflect'):
+        center=True, pad_mode='reflect', odd_n_fft=False):
```

Comment:
I'm not sure we need a separate parameter here -- there's already a parameter for frame length which can be used to specify it exactly.  Probably, if a user has a non-standard frame length, they have it in a variable somewhere, and it's easier to pass that along and then infer whether it's even or odd, rather than force the user to make that calculation externally.

Reply:
Thank you for your advice, your indication is the optimal solution. I removed `odd_n_fft` and fixed the code.

Reply:
Thanks!  The parameter is still in the call signature though.

Reply:
Sorry, I missed it. I removed unnecessary argument `odd_n_fft`.